magento/magento2
# Solved Customer Prefix not saved while guest order

### Description (*)
Customer Prefix filed in the sales_order table is not getting saved when place order as a guest customer.

### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#27849

### Manual testing scenarios (*)

1. Go To Backend Admin > Stores > Configuration > Customers > Customer Configuration > Name and Address Options and make Show Prefix Required and saved it.
2. Go to the frontend and place the order as a guest. While entering the billing address fill the Name Prefix text box.
3. After successful order placement check filed sales_order.customer_prefix in the database. It should be filled up.


### Questions or comments


### Contribution checklist (*)
